provider/pkg/gen/schema.go

```diff
@@ -199,12 +202,31 @@ func (g *packageGenerator) findResources(parent string, resources map[string]dis
 			}
 		}
 
+		typeName := fmt.Sprintf("%s%s", parent, name)
+		if override, has := overrides[g.genToken(typeName)]; has {
+			if createMethod == nil {
+				createMethod = override.retrieveCreate(res)
+			}
+			if getMethod == nil {
+				getMethod = override.retrieveGet(res)
+			}
+			if updateMethod == nil {
+				updateMethod = override.retrieveGet(res)
+			}
+			if deleteMethod == nil {
+				deleteMethod = override.retrieveDelete(res)
+			}
+		}
 		if createMethod != nil {
-			typeName := fmt.Sprintf("%s%s", parent, name)
 			err := g.genResource(typeName, createMethod, getMethod, updateMethod, deleteMethod)
 			if err != nil {
 				return err
 			}
+		} else {
+			if deleteMethod != nil || updateMethod != nil {
+				// It can be useful to look at this output and look for potential missing resources.
+				fmt.Printf("No create method for resource: %s (%s), skipping.\n", typeName, g.docName)
```

Comment:
How many of these do we have now?

Reply:
It mentioned 153 possible resources, off which 81 are variants on `Operation` types which doesn't seem to make sense to be a resource. I restricted to just those that had at least one POST method and the tally comes to 62 possible resouces:
```
No create method for resource: OrganizationApiproductAttribute (discovery/apigee_v1.json), skipping.
No create method for resource: OrganizationApiRevision (discovery/apigee_v1.json), skipping.
No create method for resource: OrganizationDeveloperAppAttribute (discovery/apigee_v1.json), skipping.
No create method for resource: OrganizationDeveloperAppKeyApiproduct (discovery/apigee_v1.json), skipping.
No create method for resource: OrganizationDeveloperAttribute (discovery/apigee_v1.json), skipping.
No create method for resource: OrganizationEnvironmentCach (discovery/apigee_v1.json), skipping.
No create method for resource: OrganizationSharedflowRevision (discovery/apigee_v1.json), skipping.
No create method for resource: AppService (discovery/appengine_v1.json), skipping.
No create method for resource: AppServiceVersionInstance (discovery/appengine_v1.json), skipping.
No create method for resource: AppService (discovery/appengine_v1beta.json), skipping.
No create method for resource: AppServiceVersionInstance (discovery/appengine_v1beta.json), skipping.
No create method for resource: RepositoryPackage (discovery/artifactregistry_v1beta1.json), skipping.
No create method for resource: RepositoryPackageVersion (discovery/artifactregistry_v1beta1.json), skipping.
No create method for resource: RepositoryPackage (discovery/artifactregistry_v1beta2.json), skipping.
No create method for resource: RepositoryPackageVersion (discovery/artifactregistry_v1beta2.json), skipping.
No create method for resource: Model (discovery/bigquery_v2.json), skipping.
No create method for resource: TransferConfigRun (discovery/bigquerydatatransfer_v1.json), skipping.
No create method for resource: TransferConfigRun (discovery/bigquerydatatransfer_v1.json), skipping.
No create method for resource: DebuggerDebuggeeBreakpoint (discovery/clouddebugger_v2.json), skipping.
No create method for resource: Group (discovery/clouderrorreporting_v1beta1.json), skipping.
No create method for resource: DeviceDeviceUser (discovery/cloudidentity_v1.json), skipping.
No create method for resource: DeviceDeviceUser (discovery/cloudidentity_v1beta1.json), skipping.
No create method for resource: Organization (discovery/cloudresourcemanager_v1beta1.json), skipping.
No create method for resource: IndexingDatasourceItem (discovery/cloudsearch_v1.json), skipping.
No create method for resource: Snapshot (discovery/compute_v1.json), skipping.
No create method for resource: ScanConfig (discovery/containeranalysis_v1alpha1.json), skipping.
No create method for resource: ScanConfig (discovery/containeranalysis_v1beta1.json), skipping.
No create method for resource: Snapshot (discovery/dataflow_v1b3.json), skipping.
No create method for resource: DatasetAnnotatedDataset (discovery/datalabeling_v1beta1.json), skipping.
No create method for resource: DatasetAnnotatedDatasetFeedbackThread (discovery/datalabeling_v1beta1.json), skipping.
No create method for resource: AnswerRecord (discovery/dialogflow_v2.json), skipping.
No create method for resource: AnswerRecord (discovery/dialogflow_v2.json), skipping.
No create method for resource: AnswerRecord (discovery/dialogflow_v2beta1.json), skipping.
No create method for resource: AnswerRecord (discovery/dialogflow_v2beta1.json), skipping.
No create method for resource: Project (discovery/firebase_v1beta1.json), skipping.
No create method for resource: DatabaseDocument (discovery/firestore_v1.json), skipping.
No create method for resource: DatabaseDocument (discovery/firestore_v1beta1.json), skipping.
No create method for resource: DatasetDicomStoreStudy (discovery/healthcare_v1.json), skipping.
No create method for resource: DatasetDicomStoreStudySeries (discovery/healthcare_v1.json), skipping.
No create method for resource: DatasetDicomStoreStudySeriesInstance (discovery/healthcare_v1.json), skipping.
No create method for resource: DatasetDicomStoreStudy (discovery/healthcare_v1beta1.json), skipping.
No create method for resource: DatasetDicomStoreStudySeries (discovery/healthcare_v1beta1.json), skipping.
No create method for resource: DatasetDicomStoreStudySeriesInstance (discovery/healthcare_v1beta1.json), skipping.
No create method for resource: BillingAccountLog (discovery/logging_v2.json), skipping.
No create method for resource: FolderLog (discovery/logging_v2.json), skipping.
No create method for resource: Log (discovery/logging_v2.json), skipping.
No create method for resource: OrganizationLog (discovery/logging_v2.json), skipping.
No create method for resource: Log (discovery/logging_v2.json), skipping.
No create method for resource: UserProject (discovery/oslogin_v1.json), skipping.
No create method for resource: UserSshPublicKey (discovery/oslogin_v1.json), skipping.
No create method for resource: UserProject (discovery/oslogin_v1alpha.json), skipping.
No create method for resource: UserSshPublicKey (discovery/oslogin_v1alpha.json), skipping.
No create method for resource: UserProject (discovery/oslogin_v1beta.json), skipping.
No create method for resource: UserSshPublicKey (discovery/oslogin_v1beta.json), skipping.
No create method for resource: CertificateAuthorityCertificateRevocationList (discovery/privateca_v1beta1.json), skipping.
No create method for resource: NamespaceRevision (discovery/run_v1.json), skipping.
No create method for resource: Revision (discovery/run_v1.json), skipping.
No create method for resource: NamespaceRevision (discovery/run_v1alpha1.json), skipping.
No create method for resource: Revision (discovery/run_v1alpha1.json), skipping.
No create method for resource: FolderSourceFinding (discovery/securitycenter_v1.json), skipping.
No create method for resource: SourceFinding (discovery/securitycenter_v1.json), skipping.
No create method for resource: ServiceDnsRecordSet (discovery/servicenetworking_v1.json), skipping.
```

Reply:
Not too bad... Anything that stands out as obviously useful? AppEngine ones?

Reply:
> Not too bad... Anything that stands out as obviously useful? AppEngine ones?

Yeah maybe appengine but nothing jumps out as a critical resource in the list - which is good.